tls_test.go

```diff
@@ -66,7 +66,7 @@ func (s *tlsServer) Serve(t *testing.T) {
 }
 
 func startTLSServer(t *testing.T, cfg *tls.Config) tlsServer {
-	l, err := tls.Listen("tcp", "127.0.0.1:0", cfg)
+	l, err := tls.Listen("tcp", "127.0.0.1:3456", cfg)
```

Comment:
I'm not sure if this is necessary, but it seems to resolve flakes with tests in Windows container.